Example/AppDelegate.swift

```diff
@@ -15,6 +16,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     lazy var carPlaySearchController: CarPlaySearchController = CarPlaySearchController()
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        
+        // Opt-in to billing per trip
+        // It’s important to call `MBXAccounts.activateSKUID(_:)` before making requests to the Directions API or the Voice API
+        // to avoid getting billed per request.
+        MBXAccounts.activateSKUID(.navigationSession)
```

Comment:
Is it necessary to call this method on launch, or is it safe for the developer to defer it until right before they make the first Directions API request? Or going in the other direction, is there an Info.plist option for activating the SKU ID? It seems like something that the developer would consider in the same breath as setting their access token.

Reply:
`.navigationSession` _doesn't_ affect the Maps SDK usage so it's safe to call before using any of the service libraries. `.navigationUser` _does_ affect the Maps SDK usage so it must be called before the Maps SDK is initialized.

I worked around this issue by activating the default (`.navigationSession`) in `+[MBXAccounts(MBXDefault) load]` so that we don't require any additional steps by default. The Info.plist option would be a good addition.